### PR TITLE
update VS2019

### LINKED ISSUE #47: Update VS2019 to latest
https://aka.ms/vs/16/release/vc_redist.x64.exe has a newer version

### PR DESCRIPTION
**Pull request by h-vetinari**
Closes #47 

Link info by resolving the redirect as follows:
```
>curl -ILSs https://aka.ms/vs/16/release/vc_redist.x64.exe | grep "Location:"
Location: https://download.visualstudio.microsoft.com/download/pr/b929b7fe-5c89-4553-9abe-6324631dcc3a/296F96CD102250636BCD23AB6E6CF70935337B1BBB3507FE8521D8D9CFAA932F/VC_redist.x64.exe
```

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
  - 2019
  - 2017
 vsver:
+ - 16
  - 16
  - 16
  - 15
 runtime_version:
+ - 14.29.30139
  - 14.29.30037
```
Comment (isuruf) [inline recipe/conda_build_config.yaml:18]: We don't need to support two versions with the same minor version. Can you remove the `14.29.30037` version?

Comment (h-vetinari) [reply recipe/conda_build_config.yaml:18]: Sure. But in turn, I don't understand why we even still have 14.28, which is also 16.{x=8}; 16.10 & 16.11 just happen to [share](https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B#Internal_version_numbering) the same runtime (minor) version.

Comment (isuruf) [reply recipe/conda_build_config.yaml:18]: 14.28 and 14.29 have different minor versions and we need both for users. See https://github.com/conda-forge/vc-feedstock/issues/36

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -11,28 +11,29 @@ vsver:
  - 16
  - 15
 runtime_version:
- - 14.29.30037
```
Comment (isuruf) [inline recipe/conda_build_config.yaml:14]: Can you add a comment (or just a URL to the issue) here about why we need different minor versions?

**Comment by isuruf** [comment]
Thanks

**Pull request merged**